A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1602,6 +1617,9 @@ var A320_Neo_UpperECAM;
             if (brakesHot) {
                 this.activeTakeoffConfigWarnings.push("brakes_hot");
             }
+            if (flapsHandle !== flapsMcdu) {
```

Comment:
```suggestion
            if (flapsMcdu !== 0 && flapsHandle !== flapsMcdu) {
```

As mentioned in another comment, I would suggest disabling the 0 case for now as 0 cannot be entered in the MCDU yet and then this new warning does not appear when starting directly on a runway.